web/src/beta/components/InsertionButton/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import InsertionButton from ".";
+
+export default {
+  component: InsertionButton,
+} as Meta;
+
+type Story = StoryObj<typeof InsertionButton>;
+
+export const Default: Story = {
+  render: () => <InsertionButton />,
```

Comment:
This line is unnecessary. If this line is written, onClick is not working.
If you want to use `render` in the future, something like `(args) => <Something {...args} />` is a basic way.